yapfify.el

```diff
@@ -52,7 +71,9 @@ If yapf exits with an error, the output will be shown in a help-window."
   (let* ((original-buffer (current-buffer))
          (original-point (point))  ; Because we are replacing text, save-excursion does not always work.
          (tmpbuf (generate-new-buffer "*yapfify*"))
-         (exit-code (yapfify-call-bin original-buffer tmpbuf)))
+         (start-line (line-number-at-pos (get-start-point)))
+         (end-line (line-number-at-pos (get-end-point)))
```

Comment:
This returns one linenumber higher than expected. I.e. Select a region ending at line 2 and this will return 3.